hnn_core/network.py

```diff
@@ -205,10 +219,22 @@ def __init__(self, params):
         self.n_cells = sum(len(self.pos_dict[src]) for src in
                            self.cellname_list)
 
-        # The legacy code in HNN-GUI _always_ defines 2 'common' and 5
-        # 'unique' feeds. They are added here for backwards compatibility
-        # until a new handling of external NetworkDrives's is completed.
-        self._add_external_feeds()
+        if initialise_hnn_drives:
```

Comment:
So: 'attaching' biases and drives as an API—WDYT?!

Reply:
This is still within the context of passing around the omnipresent `params` dict right? While I like that you maintain the current functionality of hnn-core (i.e., by allowing `Network` to inherently instantiate drives and biases by simply referencing the parameters parsed from `params` in `p_common` and `p_unique`), I'm afraid this dilutes the power of having an API that 'attaches' and 'instantiates' drives/biases in distinct stages. I actually really like the idea of 'attaching' and 'instantiating' drives separately, however, it looks like you have 1) created separate methods for each process yet 2) combined them here within the constructor.